pkg/osquery/tables/windows_associations/windows_associations.go

```diff
@@ -0,0 +1,111 @@
+// +build windows
+
+package windows_associations
```

Comment:
I wonder - does it make sense to specify in the package/table name that these are file-to-program/app associations?

Reply:
Maybe? I always feel weird when I don't name tables for the data source (in this case, `dsim`). I think I used `windows_associations` because it was simple. Maybe `dism_app_associations` or `dism_default_associations`?

@FritzX6  do you have an opinion?

Reply:
I think either of those is clearer - my concern is primarily that 'associations' is ambiguous.

Reply:
Opted for `dsim_default_associations`